cdap-app-fabric/src/main/java/io/cdap/cdap/security/auth/AuditLogSingleTopicSubscriberService.java

```diff
@@ -130,29 +126,26 @@ protected void storeMessageId(StructuredTableContext context, String messageId)
    */
   @Override
   protected void processMessages(StructuredTableContext structuredTableContext,
-                                 Iterator<ImmutablePair<String, AuditLogContext>> messages) throws Exception {
+                                 Iterator<ImmutablePair<String, AuditLogRequest>> messages) throws Exception {
 
-    Queue<AuditLogContext> auditLogContextQueue = new ArrayDeque<>();
+    List<AuditLogRequest> auditLogRequestList = new ArrayList<>();
 
     while (messages.hasNext()) {
-      ImmutablePair<String, AuditLogContext> next = messages.next();
-      AuditLogContext auditLogContext = next.getSecond();
-      if (auditLogContext.isAuditLoggingRequired()){
-        auditLogContextQueue.add(auditLogContext);
-      }
+      ImmutablePair<String, AuditLogRequest> next = messages.next();
+      auditLogRequestList.add(next.getSecond());
```

Comment:
nit: This looks like `Streams.stream(messages).map(ImmutablePair::getSecond).collect(toList())`

Reply:
The "Streams" is coming from `org.apache.curator.shaded.com.google.common.collect` 
so we want to use it ?

Reply:
I tried, it gave a classloading error.